### PR TITLE
Add data-to-graph conversion functions for library

### PR DESCRIPTION
**Pull request by themakshter**
This change adds basic versions of functions used to convert data received into
chartJS compatible data and displaying it. This has been structured into
different files with some test data for now. External libraries in the test
folder are included via CDN for now but will changed in the future.

### CODE REVIEW

File: src/test-data.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+testData = [{
```
Comment (dareid) [inline src/test-data.js]: should this be in the test directory?

File: src/main.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
+window.chartColors = {
```
Comment (dareid) [inline src/main.js]: should this be in the test directory?

**Review by dareid — CHANGES_REQUESTED** [review]

File: src/test-data.js (hunk 1 above)
Comment (themakshter) [reply src/test-data.js]: hmm, good point, let me switch them around

File: src/main.js (hunk 2 above)
Comment (themakshter) [reply src/main.js]: switching them around

**Comment by themakshter** [comment]
I have moved the files as well as removed some other codepen related stuff (like icon and title)

**Review by dareid — APPROVED** [review]

**Pull request merged**